test/cpp/microbenchmarks/bm_call_create.cc

```diff
@@ -609,6 +617,125 @@ typedef Fixture<&grpc_server_load_reporting_filter, CHECKS_NOT_LAST>
 BENCHMARK_TEMPLATE(BM_IsolatedFilter, LoadReportingFilter, NoOp);
 BENCHMARK_TEMPLATE(BM_IsolatedFilter, LoadReportingFilter, SendEmptyMetadata);
 
+// Test a filter's start_transport_stream_op_batch in isolation. Fixture 
+// specifies the filter under test (use the Fixture<> template to specify this).
+template <class Fixture>
+static void BM_StartTransportStreamOpBatch(benchmark::State& state) {
```

Comment:
One meta-comment: start_transport_stream_op_batch() is not part of call creation, so it might be better to put these tests in a different file.  You might even consider moving all of the filter benchmarks into their own file.

Reply:
Done.